create-admin.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
can we do this using php artisan instead?

Reply:
I'll figure it out sir,

Reply:
You can add menu on sidebar for that